src/components/nav-bar/NavBarMainPage.jsx

```diff
@@ -87,7 +72,7 @@ const NavBarMainPage = React.forwardRef(
               <hr className="bg-white h-p" />
               <Button
```

Comment:
ist das nicht nen <Link to="/">?

https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/

Reply:
gilt natürlich für alles folgende auch

Reply:
Das localizedNavigate oben was navigateByNavBar callt hat noch localization drin (wegen SSR braucht es ja eine page pro Sprache). Denke man kann das auch mit Gatsby Link umsetzen (evtl dann den Link mit der locale setzen) aber sollte man im Hinterkopf behalten.

Reply:
ok, was macht `setIsCollapsed` @LennartWeihs ? dann kann man's so lassen

Reply:
Der collapsed state ist für die mobile Ansicht, da gibt es dann collapsedContent (ist in Zeiler 66, also kurz hier drüber) der bei click auf das Hamburger Icon gezeigt wird - das sind hier für die NavbarMainPage genau diese Links. Weiß nicht ob man das eigentlich auch ändern wollen würde dass keine Komponenten als props übergeben werden, denke das ist hier schwieriger weil das was übergeben wird immer sehr spezifisch ist.